run_functional_tests.sh

```diff
@@ -34,9 +34,16 @@ function run_minio_server() {
     tests/functional/minio server --config-dir tests/functional/.cfg tests/functional/.d{1...4} >tests/functional/minio.log 2>&1 &
 }
 
-run_minio_server
-sleep 3
+if [ -z ${SERVER_ENDPOINT+x} ]; then
```

Comment:
I don't think we need to remove `minio` binary if already downloaded. The right way to do is to download it if not found.

Reply:
The point Is to always download fresh binary to, since you don't want to be running tests against some older binary locally.